### PR TITLE
Minutes of meeting 2022-FEB-24

### PR DESCRIPTION
**Pull request by sthagen**
Meeting preparation …

### CODE REVIEW

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company          | Role(s)                 |
+|:-----------|:----------|:-----------------|:------------------------|
+| Aditya     | Sharad    | Microsoft        | Voting Member           |
```
Comment (eddynaka) [inline meeting_minutes/220224_SARIF_TC_53.md:9]:
```suggestion
| Aditya     | Sharad    | GitHub        | Voting Member           |
```

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+| Chris      | Meyer     | Microsoft        | Voting Member           |
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| James      | Spoor     | Snyk Ltd.        | Voting Member           |
```
Comment (eddynaka) [inline meeting_minutes/220224_SARIF_TC_53.md:13]: James didn't participate in this meeting.

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+* David: Courtney uploaded a new revision in the issue per https://github.com/oasis-tcs/sarif-spec/issues/509#issuecomment-1039350887 as https://github.com/oasis-tcs/sarif-spec/files/8062707/sarif-v2.1.0-errata01-csd01-Changes.docx
+* David: Kindly asks the members to review the new revision before next meeting
+* Michael: Should we add a revision history entry in the document?
+
+## 3.3 Review current state of ecosystem ongoing work
+
+* Eddy: 
+  * GitHub provided most used languages and tools lists - that includes security and linters (anything that can produce results from users)
+  * Integration of GitHub within the IDEs per SARIF - two proof of concepts.
```
Comment (eddynaka) [inline meeting_minutes/220224_SARIF_TC_53.md:89]:
```suggestion
  * Integration of AzureDevOps/GitHub within the IDEs per SARIF - two proof of concepts.
```

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+  * Current extension was successful
```
Comment (eddynaka) [inline meeting_minutes/220224_SARIF_TC_53.md:90]:
```suggestion
  * Current PoC extension was successful and these changes will be added to the VS/VSCode extension.
```

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+* Michael: 
+  * The extension from security to also including linters has greatly extended the list of candidate projects to add SARIF export, author corresponding GitHub actions.
```
Comment (eddynaka) [inline meeting_minutes/220224_SARIF_TC_53.md:92]:
```suggestion
  * The GHAS (GitHub Advanced Security) is also including linters. Greatly extending the list of candidate projects to add SARIF export, author corresponding GitHub actions.
```

File: meeting_minutes/220224_SARIF_TC_53.md (hunk 2 above)
Comment (sthagen) [reply meeting_minutes/220224_SARIF_TC_53.md]: Thanks @eddynaka - fixed

**Review by eddynaka — APPROVED** [review]

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Proposed Teleconferences (Thursdays at 08:00 PT / 16:00 UTC for 1.5 hours)
+    ```
+    March 10
+    ```
+- Proposed Teleconferences (Thursdays at 08:00 PDT / 15:00 UTC for 1.5 hours)
```
Comment (dmk42) [inline meeting_minutes/220224_SARIF_TC_53.md:64]: "Proposed" should be "Scheduled"

**Review by dmk42 — APPROVED** [review]
Approved, after the comment has been addressed.

File: meeting_minutes/220224_SARIF_TC_53.md (hunk 6 above)
Comment (sthagen) [reply meeting_minutes/220224_SARIF_TC_53.md]: Done (and received the invitations). Thanks.

File: meeting_minutes/220224_SARIF_TC_53.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+## 5.1 End debate of other issues by 10 minutes prior to scheduled meeting end and follow the agenda from this point (Co-Chair Keaton)
+
+## 5.2 Review of Decisions Reached (Secretary Hagen)
+
+None
+
+## 5.3 Review of Action Items (Secretary Hagen)
+
+* ACTION on David: To please include an agenda item on the reporting of the two week evakuation effort
```
Comment (katrinaoneil) [inline meeting_minutes/220224_SARIF_TC_53.md:166]: evakuation?

**Review by katrinaoneil — APPROVED** [review]

File: meeting_minutes/220224_SARIF_TC_53.md (hunk 7 above)
Comment (sthagen) [reply meeting_minutes/220224_SARIF_TC_53.md]: @michaelcfanning please kindly verify that "evaluation" was the term meant ... thanks.

**Pull request merged**